### PR TITLE
Update to latest Datastream GA library

### PR DESCRIPTION
**Pull request by saimukkamala**
Updating to latest Datastream GA library
JIRA: https://cdap.atlassian.net/browse/CDAP-19595

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -133,6 +130,29 @@
...
+        <exclusion>
+          <artifactId>io.grpc</artifactId>
+          <groupId>grpc-netty</groupId>
+        </exclusion>
+      </exclusions>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.perfmark</groupId>
+      <artifactId>perfmark-api</artifactId>
```
Comment (sumitjnn) [inline pom.xml:152]: Is `perfmark-api` a requirement for datastream client?

**Review by sumitjnn — COMMENTED** [review]
I think we would need to update other tests as well e.g. `DatastreamClientTest`, `DatastreamDeltaSourceTest`

**Comment by sumitjnn** [comment]
@saimukkamala Please add Jira Id to the PR description

**Comment by saimukkamala** [comment]
>

File: pom.xml (hunk 1 above)
Comment (saimukkamala) [reply pom.xml:152]: yes this is a requirement for datastream client library

**Review by sumitjnn — APPROVED** [review]

**Pull request merged**